### ISSUE TITLE
render_book fails in Posit

### ISSUE THREAD

**Issue by rimelof** [pachterlab/LP_2021#4]
Hello,

The main paper is great, and I would like to read the supplement during a flight. Thank you for putting this resource out! I am trying to compile the book as a pdf, to read on a flight. I am having issues with render_book, both trying to output pdf_book as well as with bs4_book.

I open the Rstudio Cloud link on the preface, and then
```
setwd('supplement/')
bookdown::render_book("index.Rmd", output_format = "bookdown::bs4_book")
```

To which I get
```
Killed
Error in Rscript_render(f, render_args, render_meta, add1, add2) : 
  Failed to compile 06-text-mining.Rmd
```

Could you help me get this right?

Thanks!

**Comment by lambdamoses** [comment]
I tried to render the book on my computer and also got an error when rendering the text mining chapter, because `tidytext` now imports the `tidy` function from the `generics` package instead of `broom` and the R session couldn't find a method of `tidy` for `glm`. I added `library(broom)` and it worked. Also, there already is a PDF here, which is automatically rendered monthly along with the web version: https://github.com/pachterlab/LP_2021/blob/gh-pages/museumst.pdf

To be honest, the text is somewhat outdated, though I can't find the time to update it.
